### PR TITLE
ASUIS 230

### PR DESCRIPTION
**Pull request by duarte-daniela**
- Add new method to data pravider: searchFacultyStaffByFilters()
- Add new rest resourse: webdir_api_people_in_department

### CODE REVIEW

File: web/modules/custom/webdir_data_provider/src/Plugin/DataProvider/ElasticCloudDataProvider.php [hunk 1]
```diff
@@ -218,6 +218,58 @@ public function searchFacultyStaffByFields($query, $size, $search_fields, $exclu
...
+
+    if (!empty($title)) {
+      $filter_item = new \stdClass();
+      $filter_item->working_title = $title;
+      $filters->any[] = $filter_item;
+    }
+
+    if (!empty($expertise_areas)) {
+      $filter_item = new \stdClass();
+      $filter_item->job_campuses = $expertise_areas;
```
Comment (mlsamuelson) [inline web/modules/custom/webdir_data_provider/src/Plugin/DataProvider/ElasticCloudDataProvider.php:259]: @duarte-daniela Looks like your filter item for `expertise_areas` is overwriting the `job_campuses` filter item. Can you please fix?

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
See item requiring attention.

File: web/modules/custom/webdir_data_provider/src/Plugin/DataProvider/ElasticCloudDataProvider.php (hunk 1 above)
Comment (duarte-daniela) [reply web/modules/custom/webdir_data_provider/src/Plugin/DataProvider/ElasticCloudDataProvider.php]: done!

**Review by mlsamuelson — APPROVED** [review]
Thank you for the fix. Looks good now.

**Pull request merged**